ui/nftMint/src/main/java/com/solanamobile/mintyfresh/nftmint/MintDetails.kt

```diff
@@ -126,8 +127,10 @@ fun MintDetailsPage(
     val uiState = performMintViewModel.viewState.collectAsState().value
     val scrollBehavior = TopAppBarDefaults.exitUntilCollapsedScrollBehavior(rememberTopAppBarState())
 
-    if (uiState.mintState is MintState.Complete){
-        onMintCompleted()
+    if (uiState.mintState is MintState.Complete) {
```

Comment:
`if` should probably be inside the LaunchedEffect

Reply:
I was thinking the same but @creativedrewy explained it to me and I think it should remain